config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,23 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func isMountpointForbidden(path string) bool {
```

Comment:
WDYT about moving this logic into `ValidateMountPoints()` function?

Since we are not currently calling this function but one time (for `ValidateMountPoints`) I wonder if its necessary to separate the logic?

Reply:
Good point, have that fixed now!